tests/drivers/dma/loop_transfer/Kconfig

```diff
@@ -9,10 +9,10 @@ config DMA_LOOP_TRANSFER_CHANNEL_NR
 	int "DMA channel to use"
 	default 0
 
-config DMA_LOOP_TRANSFER_SRAM_SECTION
+config DMA_LOOP_TRANSFER_RELOCATE_SECTION
 	string "the section to place the memory buffers."
-	depends on NOCACHE_MEMORY
-	default ".nocache"
+	depends on CODE_DATA_RELOCATION
```

Comment:
- On H7, it is mentioned a non cached SRAM4 section, but there is nothing more with your chanbge made to make it non-cacheable.
I thought adding `zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM_NOCACHE) )>;`  would do, but it doesn't unfortunately.
- On F7, no non-cacheable region is available neither. So it should be done somehow once fixed on H7.